learn-arai/app/login/email-password-form.tsx

```diff
@@ -0,0 +1,81 @@
+import Link from 'next/link';
+
+import { FormEvent, useState } from 'react';
+import { FaRegUser } from 'react-icons/fa';
+import { IoMdKey } from 'react-icons/io';
+
+import Checkbox from '../components/form/checkbox';
+
+import '@/app/components/form/form.css';
+import Input from '@/app/components/form/input';
+import Submit from '@/app/components/form/submit';
+import { useAuth } from '@/app/hooks/useAuth';
+
+export const EmailPasswordForm = () => {
+    const [errorMessage, setErrorMessage] = useState<Promise<string> | null>();
+    const { signIn } = useAuth();
+
+    async function submitHandle(event: FormEvent<HTMLFormElement>) {
+        event.preventDefault();
+
+        const formData = new FormData(event.currentTarget);
+
+        const response = signIn(formData);
+        const responseStatus = (await response).status;
+        const responseMessege = (await response).message;
+
+        if ((await responseStatus) != 'success') {
```

Comment:
responseStatus is already await at line 24, you don't need to await again this line